cv_next/components/ui/cvPreview.tsx

```diff
@@ -4,7 +4,7 @@ export const CvPreview = ({ cv }: { cv: CvModel }) => {
   const previewUrl = transformToPreviewLink(cv.document_link);
 
   return (
-    <div style={{ overflow: "scroll" }}>
+    <div style={{}}>
```

Comment:
Are you sure you want to remove this overflow property?
And if you are, you can delete the entire `style` property..

Reply:
As far as I'm concerned there is a built-in scroll support for i-frame pdf reader in all browsers.
removed the style property

Reply:
Woops, I completely missed the point when we switched the preview from regular Image to iframe.